dev-resources/template/1_db.yml

```diff
@@ -25,13 +27,14 @@ Parameters:
   DBEngineVersion:
     Description: PostgreSQL engine version on RDS
     Type: String
-    Default: "12.9"
+    Default: "14.3"
```

Comment:
Also are you sure 12.9 is dead? I just swapped this in like a month ago.

Reply:
updated via https://github.com/yetanalytics/lrsql/pull/276/commits/934c4dc485c3a592c16f20a5b3bad23ace15932e